mkdocs.yml

```diff
@@ -43,4 +43,39 @@ plugins:
 nav:
 - 'Home': 'index.md'
 - 'Glossary': 'glossary.md'
+- Getting started:
+  - 'Overview': 'getting_started/overview.md'
+  - 'Projects': 'getting_started/projects.md'
+- Core Blockchain Concepts:
+  - 'Blockchain': 'core-blockchain-concepts/blockchain.md'
+  - 'Proof-of-Work (PoW)': 'core-blockchain-concepts/proof-of-work.md'
+  - Proof-of-Stake (PoS):
+    - 'Stake Enabled Height': 'core-blockchain-concepts/proof-of-stake/stake-enabled-height.md'
+    - 'Stake Valdation Height': 'core-blockchain-concepts/proof-of-stake/stake-validation-height.md'
+  - 'Chain Fork Semantics': 'core-blockchain-concepts/chain-fork-sematics.md'
+  - 'Hard Fork Voting and Consensus Rule Changes': 'core-blockchain-concepts/hard-fork-voting-and-consensus-rule-changes.md'
+  - 'Transaction Data': 'core-blockchain-concepts/transaction-data.md'
+  - 'Fork Detection': 'core-blockchain-concepts/fork-detection.md'
+  - 'Transactions': 'core-blockchain-concepts/transactions.md'
+  - 'Contracts': 'core-blockchain-concepts/contracts.md'
+  - 'Merkle root construction': 'core-blockchain-concepts/merkle-root-construction.md'
```

Comment:
Caps on "Merkle Root Contruction" to match the rest of the nav menu.

I guess things like this don't really matter right now because this PR is just putting in the skeleton of the site, but we might as well correct this now to reduce unnecessary churn later.

Reply:
Agreed. Fixed